src/Symfony/Component/Cache/Adapter/Client/MemcachedClient.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter\Client;
+
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+
+/**
+ * @author Rob Frawley 2nd <rmf@src.run>
+ */
+class MemcachedClient
+{
+    private static $serverDefaults = array(
```

Comment:
I think we don't need server defaults in the cache component (we may have in FrameworkBundle later on, but that's another topic.)